server/views/pages/recommendations/checkBookingDetails.njk

```diff
@@ -203,147 +203,18 @@
                         { key: 'Recall received time', value: formatDateTimeFromIsoString({ isoDate: recommendation.bookRecallToPpud.receivedDateTime, timeOnly: true }), actions: [{ id: editIdPrefix+'recallreceivedtime', text: editText, href: 'edit-recall-received-date-and-time', visuallyHidden: 'recall received time' }] },
                         { key: 'Recall decision date', value: formatDateTimeFromIsoString({ isoDate: recommendation.decisionDateTime, dateOnly: true }) },
                         { key: 'Recall decision time', value: formatDateTimeFromIsoString({ isoDate: recommendation.decisionDateTime, timeOnly: true }) },
-                        { key: 'Probabtion area', value: blankSafe(recommendation.bookRecallToPpud.probationArea, true, 'Enter a probation area'), actions: [{ id: editIdPrefix+'probationarea', text: editText, href: 'edit-probation-area', visuallyHidden: 'probabtion area' }], error: errors.probationArea },
+                        { key: 'Probation area', value: blankSafe(recommendation.bookRecallToPpud.probationArea, true, 'Enter a probation area'), actions: [{ id: editIdPrefix+'probationarea', text: editText, href: 'edit-probation-area', visuallyHidden: 'probation area' }], error: errors.probationArea },
                         { key: 'Probation practitioner', value: blankSafe(practitioner.name) },
                         { key: 'Probation practitioner email', value: blankSafe(practitioner.email) },
                         { key: 'Probation practitioner phone number', value: blankSafe(practitioner.telephone) },
                         { key: 'Local police contact', value: blankSafe(recommendation.bookRecallToPpud.policeForce, true, 'Enter a local police force'), actions: [{ id: editIdPrefix+'policecontact', text: editText, href: 'edit-police-contact', visuallyHidden: 'local police contact' }], error: errors.policeForce },
                         { key: 'Senior manager (ACO)', value: acoSigned.createdByUserFullName },
                         { key: 'Senior manager (ACO) email', value: acoSigned.emailAddress },
-                        { key: 'MAPPA level', value: blankSafe(recommendation.bookRecallToPpud.mappaLevel,true, 'Enter a MAPPA level'), actions: [{ id: editIdPrefix+'mpaalevel', text: editText, href: 'edit-mappa-level', visuallyHidden: 'MPAA level' }], error: errors.mappaLevel },
+                        { key: 'MAPPA level', value: blankSafe(recommendation.bookRecallToPpud.mappaLevel,true, 'Enter a MAPPA level'), actions: [{ id: editIdPrefix+'mapaalevel', text: editText, href: 'edit-mappa-level', visuallyHidden: 'MAPAA level' }], error: errors.mappaLevel },
```

Comment:
Sorry, I should've been more specific about the correction: it should be mappa/MAPPA with two Ps, not with three As.

Reply:
Thank you for spotting that.
Out of curiosity, do we have a list of acronyms like this anywhere? Or at least what MAPPA stands for? I think I have been a bit vague with getting some of these down as I have no idea what they mean so spotting whether they are right or wrong has been a bit abstract still. 😄

Reply:
Yeah, there are a couple of acronym pages I've got bookmarked: https://ministryofjustice.github.io/acronyms/ for MoJ and https://samuel-hoskin.github.io/CS-Acronyms/ is more across all of the civil service.